test/spec/modules/loglyliftBidAdapter_spec.js

```diff
@@ -0,0 +1,172 @@
+      native: {
+        body: {
+          required: true
+        },
+        icon: {
+          required: false
+        },
+        title: {
+          required: true
+        },
+        image: {
+          required: true
+        },
+        sponsoredBy: {
+          required: true
+        }
+      }
+    }
+  }];
+
+  const bidderRequest = {
+    refererInfo: {
+      referer: 'fakeReferer',
+      reachedTop: true,
+      numIframes: 1,
+      stack: []
+    },
+    auctionStart: 1632194172781,
+    bidderCode: 'loglylift',
+    bidderRequestId: '15da3afd9632d7',
+    auctionId: 'f890b7d9-e787-4237-ac21-6d8554abac9f',
+    timeout: 3000
+  };
+
+  const nativeServerResponse = {
+    body: {
+      bids: [{
+        requestId: '254304ac29e265',
+        cpm: 10.123,
+        width: 360,
+        height: 360,
+        creativeId: '123456789',
+        currency: 'JPY',
+        netRevenue: true,
+        ttl: 30,
+        meta: {
+          advertiserDomains: ['advertiserexample.com']
+        },
+        native: {
+          clickUrl: 'https://dsp.logly.co.jp/click?ad=EXAMPECLICKURL',
+          image: {
+            url: 'https://cdn.logly.co.jp/images/000/194/300/normal.jpg',
+            width: '360',
+            height: '360'
+          },
+          impressionTrackers: [
+            'https://b.logly.co.jp/sorry.html'
+          ],
+          sponsoredBy: 'logly',
+          title: 'Native Title',
+        }
+      }],
+    }
+  };
+
+  describe('isBidRequestValid', function () {
+    it('should return true if the adspotId parameter is present', function () {
+      expect(spec.isBidRequestValid(nativeBidRequests[0])).to.be.true;
+    });
+
+    it('should return false if the adspotId parameter is not present', function () {
+      let bidRequest = utils.deepClone(nativeBidRequests[0]);
+      delete bidRequest.params.adspotId;
+      expect(spec.isBidRequestValid(bidRequest)).to.be.false;
+    });
+  });
+
+  describe('buildRequests', function () {
+    it('should generate a valid single POST request for multiple bid requests', function () {
+      const request = spec.buildRequests(nativeBidRequests, bidderRequest)[0];
+      expect(request.method).to.equal('POST');
+      expect(request.url).to.equal('https://bid.logly.co.jp/prebid/client/v1?adspot_id=16');
+      expect(request.data).to.exist;
+
+      const data = JSON.parse(request.data);
+      expect(data.auctionId).to.equal(nativeBidRequests[0].auctionId);
+      expect(data.bidderRequestId).to.equal(nativeBidRequests[0].bidderRequestId);
+      expect(data.transactionId).to.equal(nativeBidRequests[0].transactionId);
+      expect(data.adUnitCode).to.equal(nativeBidRequests[0].adUnitCode);
+      expect(data.bidId).to.equal(nativeBidRequests[0].bidId);
+      expect(data.mediaTypes).to.deep.equal(nativeBidRequests[0].mediaTypes);
+      expect(data.params).to.deep.equal(nativeBidRequests[0].params);
+      expect(data.prebidJsVersion).to.equal('6.5.0-pre');
```

Comment:
This obviously makes tests fail as we move through versions.

https://github.com/prebid/Prebid.js/pull/7869

@HashimotoLogly @ChrisHuie